test/TEST-08-ISSUE-2730/test.sh

```diff
@@ -23,8 +23,11 @@ check_result_qemu() {
 }
 
 test_run() {
-    run_qemu || return 1
-    check_result_qemu || return 1
+    if run_qemu; then
+        check_result_qemu || return 1
+    else
```

Comment:
I still don't understand -- that's exactly what I intended: like the first couple of tests we don't want to fail if QEMU is not available.